src/adaptermanager.js

```diff
@@ -132,6 +132,17 @@ exports.enableAnalytics = function (config) {
   });
 };
 
+//if an adapter needs to prepare anything before the ad can be delivered, give it a chance to do so.
+exports.prepareRendering = function (doc, bidderCode) {
```

Comment:
This may prove useful to other adapters but until we have a need for it the setup Rubicon needs to work with iframes and globals can be kept in the adapter itself. Also `adaptermanager` is used when preparing to request bids and `bidmanager` is used to handle the bid responses, so preparing to render seems to belong in the later module.